docs/SGX Static Binary Demo Documentation.md

```diff
@@ -0,0 +1,25 @@
+## Intel SGX 2 Static Binary Demo
+
+This demo is designed to illustrate a static binary running in an SGX Enclave. This demo does not include attestation.
```

Comment:
```suggestion
This demo is designed to illustrate a static binary running in an SGX Enclave. This demo *does not* include attestation.
```

It might be worth it to add emphasis on this part, is attestation is otherwise a big part of Enarx.

Reply:
*does not yet*